backend/src/plugins/Automod/actions/setSlowmode.ts

```diff
@@ -0,0 +1,53 @@
+import * as t from "io-ts";
+import { automodAction } from "../helpers";
+import { convertDelayStringToMS, isDiscordRESTError, tDelayString, tNullable } from "../../../utils";
+import { LogType } from "../../../data/LogType";
+import { AnyGuildChannel } from "eris";
+
+export const SetSlowmodeAction = automodAction({
+  configType: t.type({
+    channels: t.array(t.string),
+    duration: tNullable(tDelayString),
+  }),
+
+  defaultConfig: {
+    duration: "10s",
+  },
+
+  async apply({ pluginData, actionConfig }) {
+    const duration = actionConfig.duration ? convertDelayStringToMS(actionConfig.duration)! : 0;
+
+    for (const channelId of actionConfig.channels) {
+      const channel = pluginData.guild.channels.get(channelId);
+      // 0 = Guild Text, 4 = Guild Category - Both dont allow slowmode
+      if (!channel) continue;
+      if (channel.type === 0 || channel.type === 4) continue;
```

Comment:
This seems to be reversed now, skipping channel types 0 and 4 rather than only allowing them.